tiatoolbox/dataloader/wsireader.py

```diff
@@ -0,0 +1,101 @@
+"""
+WSIReader for WSI reading or extracting metadata information from WSIs
+"""
```

Comment:
include space after module string and imports

Reply:
Updated as suggested https://github.com/TIA-Lab/tiatoolbox/commit/b49ec891c499a589134dd43e5786bd0a7d8d39ff